assets/css/style.css

```diff
@@ -257,7 +257,8 @@ pre {
   background: whitesmoke;
   outline: 1px solid lightgray;
   padding: 10px;
-  margin: 5px; }
+  margin: 1rem 5px;
```

Comment:
should we use rem for both values instead?

Reply:
good call.. swapped it out to `rem` and bumped it up just a bit to `0.5rem` for the horizontal margin